crates/miden-lib/asm/kernels/transaction/lib/tx.masm

```diff
@@ -224,7 +224,7 @@ end
 export.update_expiration_block_num
     # Ensure block_height_delta is between 1 and 0xFFFF (inclusive)
     dup neq.0 assert.err=ERR_TX_INVALID_EXPIRATION_DELTA
-    dup push.EXPIRY_UPPER_LIMIT lt assert.err=ERR_TX_INVALID_EXPIRATION_DELTA
+    dup lt.EXPIRY_UPPER_LIMIT assert.err=ERR_TX_INVALID_EXPIRATION_DELTA
```

Comment:
nit (and probably not for this PR): we should be able to use the `u32assert2 u32lt` tick here to reduce cycle count.